Gems/AtomTressFX/Code/Rendering/HairRenderObject.cpp

```diff
@@ -124,17 +124,11 @@ namespace AZ
                         streamDesc.m_elementFormat, RHI::BufferBindFlags::ShaderRead    // No need for ReadWrite in the raster fill
                     );
 
-                    m_readBuffersViews[index] = RHI::Factory::Get().CreateBufferView();
-                    RHI::ResultCode resultCode = m_readBuffersViews[index]->Init(*rhiBuffer, viewDescriptor);
-                    if (resultCode != RHI::ResultCode::Success)
-                    {
-                        AZ_Error("Hair Gem", false, "Read BufferView could not be retrieved for [%s]", streamDesc.m_bufferName.GetCStr());
-                        return false;
-                    }
+                    m_readBuffersViews[index] = aznew RHI::MultiDeviceBufferView{ rhiBuffer, viewDescriptor };
```

Comment:
Is the call to `Init()` missing here?

Reply:
No, the interface for the new `MultiDevice*View` classes is different, the constructor directly takes over the buffer and corresponding descriptor, so no additional `Init()` is needed.

Reply:
Yes, it is just storing the two parameters as members. Actual single device view creation happens later.